source-code/core/package.json

```diff
@@ -72,6 +72,8 @@
 		"zod": "^3.21.4"
 	},
 	"devDependencies": {
+		"@types/ungap__structured-clone": "^0.3.0",
+		"@ungap/structured-clone": "^1.2.0",
```

Comment:
@samuelstroschein suggested it after I raised concerns about the JSON approach. A helpful discussion can be found under "structuredClone is not defined" on stackoverflow.

Reply:
Alright. Maybe we should run the tests of ide-extension dependencies against node 16 as well :)

Reply:
I voted for polyfilling instead of JSON.parse(JSON.stringify()) because core needs node18. 

The IDE extension is an exception. Making sure that core works for node 16 has ecosystem wide impact which seems to outweigh the alternative of polyfilling in the only app that runs on node 16, the ide extension.

Reply:
Yes, I understand this. Probably we should test the ide-extension dependencies to run on node 16.

Reply:
Sorry, misunderstanding. Ofc `core` should not polyfill! This line of code is a big time mistake. See https://github.com/inlang/inlang/pull/1051#discussion_r1259468861

Reply:
Fixed!